### ISSUE TITLE
User Activity report doesn't show unique users when all websites are selected

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#5326]
http://vladtest48e.admin.kademi-ci.co/reporting/userVisitReport
![image](https://user-images.githubusercontent.com/1945230/38733537-ddb80638-3f2a-11e8-9555-399bd0e03355.png)

However when you specify website - it show users successfully.

**Comment by anhnt** [comment]
Yup its expected because unique users are being counted per website. I dont think we count for all websites
